build.gradle.kts

```diff
@@ -26,41 +34,42 @@ dependencies {
   implementation("org.springframework.boot:spring-boot-starter-oauth2-client")
   implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
   implementation("org.springframework.boot:spring-boot-starter-mail")
-  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:1.3.0")
+  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:2.1.1")
 
-  implementation("org.springdoc:springdoc-openapi-ui:1.7.0")
-  implementation("org.springdoc:springdoc-openapi-data-rest:1.7.0")
-  implementation("org.springdoc:springdoc-openapi-kotlin:1.7.0")
-  implementation("org.springdoc:springdoc-openapi-webmvc-core:1.7.0")
+  implementation("org.springdoc:springdoc-openapi-starter-webmvc-api:2.3.0")
```

Comment:
Better to remove the version number here, then they are set automatically by the Spring dependency manager so all versions will be aligned correctly.

Reply:
Oh sorry this is springdoc not springframework - the version of this isn't managed by Spring dependency manager, I blatantly lied. Sorry!